db/sql/migrations/add.sql

```diff
@@ -0,0 +1,10 @@
+/*
+  DESCRIPTION: Adds a new record into the migrations table
+
+  PARAMETERS:
+  	- id: record id
+  	- name: name of the /update file that did the migration
+*/
+
+INSERT INTO migrations(id, name)
+VALUES(${id}, ${name}) ON CONFLICT DO NOTHING
```

Comment:
`ON CONFLICT DO NOTHING` is not recommended because every particular migration should be executed and then inserted to this table only once.

Reply:
This PR wasn't about semantics of the SQL queries. This one was taken exactly from the code. If there is an issue with this  query, it should be addressed separately ;)

Reply:
From decription: `This is a complete rewrite from scratch of the entire database migration layer` :D

Reply:
Yes, but changing the logic of queries is to change how the migration logic. I didn't change the migration logic, only implementation of that logic :wink:

Reply:
@4miners If you are aware of any problems with migrations that need to be addressed - please, open an issue for it, and I will look into it! ;)